### PR TITLE
133  - Set permissions

### LINKED ISSUE #133: Set permissions
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by AGorinam**
closes #133

### CODE REVIEW

File: app/policies/organization_policy.rb [hunk 1]
```diff
@@ -10,27 +10,28 @@ def new?
...
-    record.users.include?(user)
+  def show?
+    true
   end
 
-  def create?
-    true
+  def edit?
+                  # super_team_member is a team_member
+    user.admin? || user.is_team_member_of(record) && user.is_super_team_member?
```
Comment (christinasugiono) [inline app/policies/organization_policy.rb:23]: Do any of the conditions need to be wrapped in parentheses? Or is it possible to replace the `user.is_team_member_of(record) && user.is_super_team_member?` with `user_is_super_team_member?` 🤔

File: app/policies/application_policy.rb [hunk 2]
```diff
@@ -53,11 +53,17 @@ def resolve
 
   private
 
-  def user_is_owner_or_admin?
-    record.user == user || user.admin? if !user.nil? || record.organization.organization_owner?(user)
+  def user_is_super_team_member?
```
Comment (christinasugiono) [inline app/policies/application_policy.rb:56]: I have a question, do we need something like this for the user as admin as well? Check both if they're admin and a member of the team?

Comment (AGorinam) [reply app/policies/application_policy.rb:56]: from what I think, admin will be us (god user) and then we have super_user which will be each super_team_member (organization admin? lmk if that's what you were talking about

File: app/policies/organization_policy.rb (hunk 1 above)
Comment (AGorinam) [reply app/policies/organization_policy.rb:23]: the think is that we have to check that the user has a super_team_member role and that it is part of an organization (record). Maybe we can think of a way to make it more clear and easy to understand (changing the schema and role management)

**Pull request merged**